src/task/ReceiveVideoStreamIndexTask.ts

```diff
@@ -223,6 +223,16 @@ export default class ReceiveVideoStreamIndexTask
 
     // Intersect `this.context.videoSendCodecPreferences` with `index.supportedReceiveCodecIntersection`
     for (const capability of this.context.videoSendCodecPreferences) {
+      let isCapabilityBlocklisted = false;
+      for (const blocklistedCapability of this.context.videoSendCodecsBlocklisted) {
```

Comment:
nit: Wonder if there is a better name than `videoSendCodecsBlocklisted` ? Blocklist seems a bit strong and sounds like a setting someone set.
Can we just say `videoSendCodecsCauseDegration`

Reply:
agree, maybe `degradedVideoSendCodecs`/`brokenVideoSendCodecs`/etc?

Reply:
Updated the name to `degradedVideoSendCodecs`